poc/test_sigma_protocols.sage

```diff
@@ -10,23 +10,23 @@ import json
 
 def test_vector(test_vector_function):
     def inner(vectors, suite):
-        NIZK = CIPHERSUITE[suite]
+        NISigmaProtocol = CIPHERSUITE[suite]
```

Comment:
nit: Perhaps `NISigmaProtocolShortProof` to distinguish from batchable proofs? Otherwise, if you want to express that short proofs are the default, perhaps change the tag to "NISigmaProtocol". (I think this would be reasonable, FWIW -- the default should be the cheap, hard-to-misuse thing.)

Reply:
I just shortened the tag to one byte, 0x"AA" and 0x"BB", which I think addresses the constant-size-tag and prefixing concerns. Happy to change the exact byte, but I do think that a one-byte tag is better on multiple fronts.